stdlib/public/core/VarArgs.swift

```diff
@@ -122,7 +122,7 @@ public func _encodeBitsAsWords<T : CVarArg>(_ x: T) -> [Int] {
 }
 
 // CVarArg conformances for the integer types.  Everything smaller
-// than a CInt must be promoted to CInt or CUnsignedInt before
+// than a Int32 must be promoted to Int32 or CUnsignedInt before
```

Comment:
Comment change: "a" to "an". Also, probably worth doing CUnsignedInt and the rest at the same time?